INSTALL.md

```diff
@@ -177,7 +177,24 @@ sudo service replicated-operator restart
 
 ### Backups
 
-Waffle Takeout automatically takes snapshots of the application state and database every 24 hours. These snapshots can be used to restore your Takeout installation in the event that something happens to the host machine or the installation itself. Snapshots are stored in `/var/lib/replicated/snapshots` on the host machine. It is highly recommended that you copy these backups to another location each day so that you are not at risk of losing them along with your installation in the event the host machine encounters complications.
+Snapshots can be used to restore your Takeout installation in the event that 
+something happens to the host machine or the installation itself. Snapshots are
+stored in `/var/lib/replicated/snapshots` on the host machine. It is highly
+recommended that you copy these backups to another location on a regular basis
+to minimize risk of losing them along with your installation in the event the
+host machine encounters complications.
+
+**This must be manually configured using the admin console.  To do so, follow
+these steps:**
+1. Go to the management interface on port 8800
```

Comment:
Seems like mardown isn't rendering correctly. Might need to add a newline break after "follow these steps:"?